docs/calibration.md

```diff
@@ -50,3 +50,14 @@ agipd_cd = CalibrationData.from_report(3757)
 ::: extra.calibration.JUNGFRAUConditions
 
 ::: extra.calibration.LPDConditions
+
+## Bad pixel values
+
+The calibration pipeline produces masks along with corrected data, in keys
+called `image.mask` or `data.mask` depending on the detector. Zeros in the mask
+represent good pixels, while any other value indicates one or more reasons to
+consider the corresponding data dubious:
```

Comment:
I would prefer as neutral wording as possible, given some bits simply mean to inform about a _peculiarity_ like `NON_STANDARD_SIZE`. Maybe conspicuous? (just grabbing that from a dictionary)

Reply:
Good point, I've had a go at rewording this.

Are you happy with the subheading remaining 'Bad pixel values' if we give more nuance in the description,  or do you think that needs to change too?

Reply:
I suspect that term is just too established to change now 🤷